package.json

```diff
@@ -15,12 +15,18 @@
   },
   "dependencies": {
     "axios": "^0.21.1",
+    "bootstrap": "^4.6.0",
     "framer-motion": "^3.2.1",
     "next": "10.0.4",
+    "node": "^15.10.0",
     "prop-types": "^15.7.2",
     "react": "17.0.1",
     "react-dom": "17.0.1",
-    "sass": "^1.30.0"
+    "react-hook-form": "^6.15.5",
+    "react-modal": "^3.12.1",
+    "reactstrap": "^8.9.0",
```

Comment:
We don't need this

Reply:
reactstrap is unnecessary and will be removed